Chummer/Classes/ColorManager.cs

```diff
@@ -224,6 +224,22 @@ private static Color TransformToDarkModeValidVersion(Color objColor)
         private static Color GrayHasNotesColorDark => GenerateDarkModeColor(GrayHasNotesColorLight);
         public static Color ErrorColor => Color.Red;
 
+        public static Color DieGlitchFore => IsLightMode ? DieGlitchForeLight : DieGlitchForeDark;
+        public static Color DieGlitchBackground => IsLightMode ? DieGlitchBackgroundLight : DieGlitchBackgroundDark;
+        public static Color DieHitFore => IsLightMode ? DieHitForeLight : DieHitForeDark;
+        public static Color DieHitBackground => IsLightMode ? DieHitBackgroundLight : DieHitBackgroundDark;
+
+        private static Color DieGlitchForeLight => Color.White;
+        private static Color DieGlitchBackgroundLight => Color.DarkRed;
+        private static Color DieHitForeLight => Color.Black;
+        private static Color DieHitBackgroundLight => Color.LightGreen;
+
+        private static Color DieGlitchForeDark => GenerateDarkModeColor(Color.White);
+        private static Color DieGlitchBackgroundDark => GenerateDarkModeColor(Color.DarkRed);
+        private static Color DieHitForeDark => GenerateDarkModeColor(Color.Black);
+        private static Color DieHitBackgroundDark => GenerateDarkModeColor(Color.LightGreen);
```

Comment:
If you want to cut down on code duplication, these should probably refer to the Light mode colors through their properties instead of directly to the color to which you set those properites.